eva_submission/eload_brokering.py

```diff
@@ -233,6 +235,43 @@ def _ena_report(self):
     - receipt: {receipt}
 """.format(**report_data))
         return '\n'.join(reports)
+    
+    def _archival_confirmation_text(self):
+        metadata_file = self.eload_cfg.query('submission', 'metadata_spreadsheet')
+        reader = EvaXlsxReader(metadata_file)
+        study_title = reader.project_title
+
+        hold_date = self.eload_cfg.query('brokering', 'ena', 'hold_date')
+        brokering_date = self.eload_cfg.query('brokering', 'brokering_date')
+        brokering_date_plus_3 = datetime.datetime.strptime(brokering_date.split(" ")[0].strip(), "%Y-%m-%d").date() + datetime.timedelta(days=3)
+        available_date = hold_date if hold_date is not None else brokering_date_plus_3
```

Comment:
Considering that the source of truth is currently ENA, We should be checking ENA for the date in case it's been updated. Unfortunately, we can't get it from the API unless the project is public. This will have to do for now.
cc @BaronKoy